### DISCUSSION TITLE
Enable/Disable Alerts via MQTT/API

### DISCUSSION THREAD

**Discussion by TFenby** [blakeblackshear/frigate#15301] [Ideas]
This would be handy as a sort of arm/disarm. For example, I'd like to have my back yard camera to _detect_ people at all times (for easy review/scrubbing of video), but if I'm away from home, my alarm system is armed, etc., I'd like it also to _alert_ on people.

**Comment by hawkeye217** [comment]
There's a pinned feature request for that here: https://github.com/blakeblackshear/frigate/issues/13038

**Reply by TFenby** [reply]
Ah, I searched discussions but not issues! Thanks!
